lms/envs/devstack.py

```diff
@@ -371,6 +371,9 @@ def should_show_debug_toolbar(request):
 if FEATURES['ENABLE_ENTERPRISE_INTEGRATION']:
     SYSTEM_WIDE_ROLE_CLASSES.extend(['enterprise.SystemWideEnterpriseUserRoleAssignment'])
 
+# List of enterprise customer uuids to exclude from transition to use of enterprise-catalog
+ENTERPRISE_CUSTOMERS_EXCLUDED_FROM_CATALOG = ()
```

Comment:
In common.py above, `ENTERPRISE_CUSTOMERS_EXCLUDED_FROM_CATALOG` is assigned `[]` whereas here (and the default below) it's assigned `()`. Is this discrepancy intentional?

Reply:
yeah I was wondering the same thing... like I mentioned I just followed the same pattern used by CORS_ORIGIN_WHITELIST. My guess is that common.py imports it as a list since it'll be changed by one of the settings files that inherits from it, and it's imported as a tuple in other places to prevent subsequent changes (we don't want our list of enterprises to be mutable in the end)